### PR TITLE
Feature/rxjava2

### PR DESCRIPTION
**Pull request by Kritarie**

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -1,3 +1,4 @@
+include ':redux-rx2'
 rootProject.name = 'redux'
```
Comment (evant) [inline settings.gradle:1]: Should keep this at the top

**Review by evant — COMMENTED** [review]
Please squash your commits. Also a Maybe dispatcher would be good.

**Comment by Kritarie** [comment]
@evant done

**Comment by evant** [comment]
TODO: need to update README with new dep

**Pull request merged**